### DISCUSSION TITLE
Welcome to The-Anki-Vector Discussions!

### DISCUSSION THREAD

**Discussion by Damantha126** [Damantha126/The-Anki-Vector#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Sadew451** [comment]
ammo

**Reply by Damantha126** [reply]
why

**Reply by PercyOfficial** [reply]
Halo

**Reply by Damantha126** [reply]
Hi

**Comment by Sadew451** [comment]
Hi
